libs/coin-framework/package.json

```diff
@@ -115,6 +115,7 @@
   "scripts": {
     "clean": "rm -rf lib lib-es",
     "build": "tsc && tsc -m esnext --moduleResolution bundler --outDir lib-es",
+    "coverage": "jest --coverage --passWithNoTests && mv coverage/coverage-final.json coverage/coverage-coin-framework.json",
```

Comment:
What do you think of putting all those configuration parameters in the `jest.config.js` ?